OpenMEEG/include/filenames.h

```diff
@@ -37,12 +37,19 @@ namespace OpenMEEG {
     }
 
     inline std::string
+    #ifdef _MSC_VER
+    #pragma warning( push )
+    #pragma warning( disable:4244 )  /*  warning C4244: '=': conversion from 'int' to 'char', possible loss of data */
```

Comment:
This one seemed too difficult to solve but @papadop if it's clear to you feel free to push